ijson: not packaged in debian
nightly: incompatible with release build

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -33,4 +33,0 @@ repository = "https://github.com/timothe
-[dependencies.ijson]
-version = "^0.1"
-optional = true
-
@@ -53,4 +45,2 @@ all-impls = [
     "smallvec",
-    "serde_json",
-    "ijson",
 ]
@@ -61,3 +51,2 @@ default = [
 ]
-nightly = []
 std = []
